### PR TITLE
Do not skip invalid TODOs

### LINKED ISSUE #2733: Make linter for checking that linked issues are open
### What should be done?

We have a linter that checks comments like `// TODO https://github.com/FerretDB/FerretDB/issues/<number>` in our Go code: https://github.com/FerretDB/FerretDB/tree/main/tools/checkcomments

It should also check if those issues are still open. Doing that is a bit tricky: the tool runs once for each Go package (not project or Go module) and generates so many requests (many of which are for the same issues) that we easily go over GitHub's rate limit. We should keep cache check results in a temporary file and coordinate multiple invocations using [`lockedfile`](https://github.com/rogpeppe/go-internal/tree/master/lockedfile).

Additionally (and in a separate PR), we should check those issue links in string constants like in
```go
skip: "https://github.com/FerretDB/FerretDB/issues/1522"
```

### Where?

https://github.com/FerretDB/FerretDB/tree/main/tools/checkcomments

### Definition of Done

* Linter does not check the same issue twice for one `task lint` invocation.
* String constants in Go code are checked.

### PR DESCRIPTION
**Pull request by AlekSi**
# Description

Closes #2733.

## Readiness checklist

- [x] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [x] I added/updated comments and checked rendering.
- [x] I made spot refactorings.
- [ ] I updated user documentation.
- [x] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: internal/handlers/common/validate.go [hunk 1]
```diff
@@ -52,10 +52,8 @@ func Validate(ctx context.Context, msg *wire.OpMsg, l *zap.Logger) (*wire.OpMsg,
 			"ns", db+"."+collection,
 			"nInvalidDocuments", int32(0),
 			"nNonCompliantDocuments", int32(0),
-			"nrecords", int32(-1), // TODO
+			"nrecords", int32(-1),
```
Comment (chilagrow) [inline internal/handlers/common/validate.go:55]: This seems like it needs todo issue or remove completely.

File: internal/handlers/sqlite/msg_drop.go [hunk 2]
```diff
@@ -78,7 +78,7 @@ func (h *Handler) MsgDrop(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, er
 		var reply wire.OpMsg
 		must.NoError(reply.SetSections(wire.OpMsgSection{
 			Documents: []*types.Document{must.NotFail(types.NewDocument(
-				"nIndexesWas", int32(1), // TODO
```
Comment (chilagrow) [inline internal/handlers/sqlite/msg_drop.go:81]: Do we need an issue for this?

**Review by rumyantseva — APPROVED** [review]
:shipit:

File: integration/create_test.go [hunk 3]
```diff
@@ -30,7 +30,7 @@ import (
 )
 
 func TestCreateStress(t *testing.T) {
-	// TODO rewrite using teststress.Stress
+	// It should be rewritten to use teststress.Stress.
```
Comment (noisersup) [inline integration/create_test.go:33]: Don't we want to make issues for those?

**Review by noisersup — DISMISSED** [review]
LGTM

**Review by rumyantseva — DISMISSED** [review]
LGTM

File: integration/create_test.go (hunk 3 above)
Comment (AlekSi) [reply integration/create_test.go:33]: No, that just needs to be done when we change that test the next time

**Comment by AlekSi** [comment]
Merged to unblock all other PRs

**Pull request merged**